docs/tasks.md

```diff
@@ -241,7 +241,24 @@ steps:
     #!/usr/bin/env bash
     /bin/my-binary
 ```
+#### Specifying a timeout
```

Comment:
thanks for adding docs! could you add a link to this in the TOC at the top of https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md ?